rpcserver.go

```diff
@@ -494,64 +494,46 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 
 	var seedlingMeta *proof.MetaReveal
 
-	// If a custom decimal display is set, the meta type must also be set to
-	// JSON.
+	// If a custom decimal display is set, we require the AssetMeta to be
+	// set. That means the user has to at least specify the meta type.
 	if req.Asset.DecimalDisplay != 0 && req.Asset.AssetMeta == nil {
-		return nil, fmt.Errorf("decimal display requires JSON asset " +
+		return nil, fmt.Errorf("decimal display requires asset " +
 			"metadata")
 	}
 
+	// Decimal display doesn't really make sense for collectibles.
+	if req.Asset.DecimalDisplay != 0 &&
```

Comment:
So we'll reject this on RPC, but still allow the possibility on the proof/protocol level?

Reply:
Yeah. We only validate the meta data at creation time. At proof validation time we don't validate anything other than the hash of the reveal matching the committed meta hash.
And IMO that's probably okay, as the meta data fields aren't necessarily things we need to enforce at the protocol level.